setup.py

```diff
@@ -4,9 +4,8 @@
 import sys
 from pathlib import Path
 
-from setuptools import find_packages, setup
-
 from qpc.release import AUTHOR, AUTHOR_EMAIL, ENTRYPOINT, PKG_NAME, URL, VERSION
+from setuptools import find_packages, setup
```

Comment:
huh, this is weird. qpc should be in the last section of imports 🤔

Reply:
ah, we need a ruff section for isort, configuring qpc as known first party!

Reply:
https://beta.ruff.rs/docs/settings/#known-first-party

Reply:
Ahh, thank you.  I'll give that a try.